### PR TITLE
dep: rm unused dependencies

### PR DESCRIPTION
**Pull request by tcoratger**

### CODE REVIEW

File: crates/consensus/beacon/Cargo.toml [hunk 1]
```diff
@@ -78,5 +76,5 @@ optimism = [
     "reth-primitives/optimism",
     "reth-provider/optimism",
     "reth-blockchain-tree/optimism",
-    "reth-rpc/optimism",
+    # "reth-rpc/optimism",
```
Comment (mattsse) [inline crates/consensus/beacon/Cargo.toml:79]: can be removed?

**Review by mattsse — COMMENTED** [review]
ty, one q

File: crates/consensus/beacon/Cargo.toml (hunk 1 above)
Comment (tcoratger) [inline crates/consensus/beacon/Cargo.toml:79]:
```suggestion
```

Comment (tcoratger) [reply crates/consensus/beacon/Cargo.toml]: oups done

**Review by mattsse — APPROVED** [review]

**Review by onbjerg — APPROVED** [review]
lgtm pending conflicts

**Pull request merged**